Source/Scene/Scene.js

```diff
@@ -2046,20 +2043,33 @@ define([
             shadowMap.update(frameState);
 
             if (!shadowMap.outOfView && defined(globe) && globe.castShadows) {
+                // PERFORMANCE_TODO: We update the globe for each face of the shadow cube map.
+                // We could create some type of camera representing a point light.
```

Comment:
As we talked about, I think this will be the way we have to go. `shadowMapCamera` could probably fake it.